Automattic/wp-calypso
# Stats: Add All Time Views Summary

Closes #1960 

This branch adds the "All Time Views" table to Calypso:

<img width="929" alt="stats_ _fly_fishers_place_ _wordpress_com" src="https://cloud.githubusercontent.com/assets/22080/22913310/c869276c-f21e-11e6-90bb-f2cbdf38783a.png">

The design was provided by @folletto in the [issue](https://github.com/Automattic/wp-calypso/issues/1960#issuecomment-276680098) - and I think it looks great!  The page is currently linked from the "Views" tab on the All-Time component seen on the Insights page:

<img width="931" alt="stats_ _fly_fishers_place_ _wordpress_com" src="https://cloud.githubusercontent.com/assets/22080/22913338/fcf94fde-f21e-11e6-804d-2b5a50c50940.png">

In Atlas stats, this page was linked via a text "Summary" link shown above the chart.  So I'm not certain if the current link strategy is discoverable enough.

__Notes__
The old Summary page also had a listing of "Recent Week" view counts.  We could explore adding this feature in a future PR, or possibly using the "Summary Bar" which would allow users to see view tables for Last 7, 30, 90, 365 / All-Time.

Should we make the `<Card />` the table is contained in horizontally scroll on mobile?

__To Test__
- Access the new table via the Stats Insights page
- Click on Year/Month and individual cells to see the full values ( if abbreviated )
